debugcc.h

```diff
@@ -43,6 +43,9 @@ struct debug_mux {
 	const char *block_name;
 	size_t size;
 
+	struct debug_mux *parent;
+	unsigned long mux;
```

Comment:
Nit: I feel like we should rename this to a perhaps-more-clear `parent_mux_value`? And perhaps have some doc-comments on the members?